ecosystem/sep-0006.md

```diff
@@ -388,9 +396,9 @@ Name | Type | Description
 `type` | string | Type of withdrawal. Can be: `crypto`, `bank_account`, `cash`, `mobile`, `bill_payment` or other custom values. This field may be necessary for the anchor to determine what KYC information is necessary to collect.
 `dest` | string | The account that the user wants to withdraw their funds to. This can be a crypto account, a bank account number, IBAN, mobile number, or email address.
 `dest_extra` | string | (optional) Extra information to specify withdrawal location. For crypto it may be a memo in addition to the `dest` address. It can also be a routing number for a bank, a BIC, or the name of a partner handling the withdrawal.
-`account` | `G...` string | (optional) The stellar account ID of the user that wants to do the withdrawal. This is only needed if the anchor requires KYC information for withdrawal. The anchor can use `account` to look up the user's KYC information. Note that the account specified in this request could differ from the account authenticated via SEP-10.
-`memo` | string | (optional) A wallet will send this to uniquely identify a user if the wallet has multiple users sharing one Stellar account. The anchor can use this along with `account` to look up the user's KYC info.
-`memo_type` | string | (optional) Type of `memo`. One of `text`, `id` or `hash`.
+`account` | `G...` or `M...` string | (optional) The Stellar or muxed account the client will use as the source of the withdrawal payment to the anchor. If SEP-10 authentiation is not used, the anchor can use `account` to look up the user's KYC information. Note that the account specified in this request could differ from the account authenticated via SEP-10.
+`memo` | string | (optional) This field should only be used if SEP-10 authentication is not. It was originally intended to distinguish users of the same Stellar account. However if SEP-10 is supported, the anchor should use the `sub` value included in the decoded SEP-10 JWT instead. See the [Shared Account Authentication](#shared-omnibus-or-pooled-accounts) section for more information.
```

Comment:
The anchor should associate the account/memo/muxed account included in the `sub` field with the KYC information collected, but the stellar account used as the source of funds withdrawn can be different than the `sub` value.

If SEP-10 is not supported though, then the `account` parameter cannot be different than the source of funds withdrawn. This is more of an edge case since any service that collects KYC needs to implement SEP-10 to ensure the KYC data provided can only be modified by the initial provider. 

Ideally, we make a breaking change and require SEP-10 for SEP-6 implementations so we don't have to make this distinction and can remove the potential for confusion.

Reply:
Oh I see – I didn't realize this was mostly focused on the fact SEP-10 is not a requirement for SEP-6. So it seems like our options to simplify this are either:

a) as you suggested, we make SEP-10 a requirement for SEP-6 (which is a breaking change); or
b) we change the entire SEP to require that `account == source of withdrawn funds`, which is also breaking.

I'm not comfortable with making a breaking change at this point, so let's keep this line as-is.